src/components/BackButton/BackButton.tsx

```diff
@@ -0,0 +1,26 @@
+import { StyledBackButton } from './StyledBackButton';
+import { ButtonProps } from '../Button/Button';
+import { useNavigate } from 'react-router';
+
+export interface BackButtonProps extends ButtonProps {
```

Comment:
See previous comment

```suggestion
export interface BackButtonProps extends ButtonHTMLAttributes<HTMLButtonElement> {
```